packages/react-context-selector/config/tests.js

```diff
@@ -0,0 +1,7 @@
+/** Jest test setup file. */
+
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+// Configure enzyme.
+configure({ adapter: new Adapter() });
```

Comment:
I don't think that we are using Enzyme in this package, can you please check?

Reply:
I've removed enzyme from devdeps and will remove this too